crates/sui-framework/src/lib.rs

```diff
@@ -152,13 +152,13 @@ pub fn run_move_unit_tests(
 /// Wrapper of the build command that verifies the framework version. Should eventually be removed once we can
 /// do this in the obvious way (via version checks)
 pub fn build_move_package(path: &Path, config: BuildConfig) -> SuiResult<CompiledPackage> {
-    let test_mode = config.config.test_mode;
+    //let test_mode = config.config.test_mode;
     let pkg = config.build(path.to_path_buf())?;
-    if test_mode {
+    /*if test_mode {
```

Comment:
Uncomment?

Reply:
Good call--I just did this because it's impossible to build/test w/ framework changes otherwise :(